tools/telemetry-generator/src/handlers/executionTimeTestHandler.ts

```diff
@@ -18,8 +18,8 @@ module.exports = function handler(fileData, logger): void {
 			benchmarkType: "ExecutionTime",
 			suiteName: fileData.suiteName,
 			benchmarkName: testData.benchmarkName,
-			arithmeticMean: testData.stats.arithmeticMean,
-			marginOfError: testData.stats.marginOfError,
+			arithmeticMean: testData.customData["period (ns/op)"],
```

Comment:
I believe we changed the capitalization of this key in benchmark tools, so we need to make sure we match it here. Let's double check all keys to make sure they have the exact same capitalization than how we output them to the files.

Reply:
This is a great catch. I did indeed make capitalization changes to both types of tests, so @chentong7 please make sure to apply the changes 😄 .